website/docs/developer/research/research-deep-glacier-migration.mdx

```diff
@@ -157,16 +139,20 @@ for retrieve_object in retrieve_objects:
         print(f"{ex} for {retrieve_object}")
 
 ```
-Output:
+The resultant output from the boto3 `restore_object` command can bee seen below:
 ```
 {'ResponseMetadata': {'RequestId': '61ZWC27NHSFNMYJT', 'HostId': 'ItKZaYvn9rfi6ZVYmUjfZX24mKJlDZlciT9MaKK0aHiNuLWB3Pt8WSEMNt5yBK+5u+MJfzjFDfU=', 'HTTPStatusCode': 202, 'HTTPHeaders': {'x-amz-id-2': 'ItKZaYvn9rfi6ZVYmUjfZX24mKJlDZlciT9MaKK0aHiNuLWB3Pt8WSEMNt5yBK+5u+MJfzjFDfU=', 'x-amz-request-id': '61ZWC27NHSFNMYJT', 'date': 'Sun, 24 Jul 2022 18:01:33 GMT', 'server': 'AmazonS3', 'content-length': '0'}, 'RetryAttempts': 0}}
 {'ResponseMetadata': {'RequestId': 'PNN4XXA5SYTBBRKG', 'HostId': 'HgiIiqfb0hZPHgZml98VCyLLfD5LyQ8pSLpgGN6hwUUMtmcPZoo/ACbCL1rXz+pXZ4Ce2UEe34s=', 'HTTPStatusCode': 202, 'HTTPHeaders': {'x-amz-id-2': 'HgiIiqfb0hZPHgZml98VCyLLfD5LyQ8pSLpgGN6hwUUMtmcPZoo/ACbCL1rXz+pXZ4Ce2UEe34s=', 'x-amz-request-id': 'PNN4XXA5SYTBBRKG', 'date': 'Sun, 24 Jul 2022 18:01:34 GMT', 'server': 'AmazonS3', 'content-length': '0'}, 'RetryAttempts': 0}}
 ```
+Another option is to look into [S3 batch operations](https://docs.aws.amazon.com/AmazonS3/latest/userguide/batch-ops-initiate-restore-object.html). It can perform a single operation on lists of Amazon S3 objects and a single job can perform a specified operation on billions of objects containing exabytes of data.
 
-Also see existing ORCA [restore_object function](https://github.com/nasa/cumulus-orca/blob/develop/tasks/request_files/request_files.py#L548)
+3. Once bulk retrieval of the files is complete, typically 5 - 12 hours from the request time, the bucket event notification for `ObjectRestore:Completed` action will trigger code that will copy the restored object back to the ORCA bucket as a DEEP ARCHIVE storage type.  The code will then update the ORCA catalog storage type for the object to `Deep Archive` and the migration job status. Note that the current existing restore workflow trigger should be disabled during the whole migration process to prevent triggering the `post_copy_request_to_queue` lambda and then re-enabled when migration is complete.
+S3 batch operation can also be used in this case to perform the copying. For additional information, see https://docs.aws.amazon.com/AmazonS3/latest/userguide/batch-ops-copy-object.html.
+
+Another option could be reusing the [ORCA recovery workflow](https://nasa.github.io/cumulus-orca/docs/about/architecture/architecture-recover-container) tasks and code to perform the migration. Additional research should be done prior to implementation of this use case to better validate the potential reuse of the recovery workflow for this task.
 Make sure the S3 bucket policy has permission for `s3:PutObject*` action otherwise user will see access denied error. See this [policy](https://github.com/nasa/cumulus-orca/blob/develop/website/docs/developer/deployment-guide/creating-orca-glacier-bucket.md#archive-bucket) for additional details.
```

Comment:
This needs to be separate paragraph with some preface sentences. Maybe something like the following:
```
Additional items to consider when implementing this step are the bucket policy and the use of `boto3.copy_object`. Make sure ...
```